05-cluster-prerequisites.md

```diff
@@ -35,7 +35,17 @@ Following the steps below will result in the provisioning of the shared Azure re
+  > :book: The app team is about to provision three shared Azure resources. One is a non-regional and two regional, but more importantly they are deployed independently from their AKS Clusters:
+  >
+  > | Azure Resource                                                                                                | Non-Regional  |  East US 2  | Central US |
+  > |---------------------------------------------------------------------------------------------------------------|:-------------:|:-----------:|:----------:|
+  > | [Log Analytics in Azure Monitor](https://docs.microsoft.com/azure/azure-monitor/logs/log-analytics-overview)  |               |      ✓      |            |
+  > | [Azure Container Registry](https://docs.microsoft.com/azure/container-registry/)                              |               |      ✓      |      ✓     |
+  > | [Azure Front Door](https://docs.microsoft.com/azure/frontdoor/front-door-overview)                            |      ✓        |             |            |
+  >
+  > **Azure Monitor logs solution**
+  >
+  > The app team is creating multiple clusters for its new workload (Application Id: a0042). This array of clusters is a multi-region infrastructure solution composed by multiple Azure resources that regularly emit logs to Azure Monitor. All the collected data is stored in a [centralized Log Analytics workspace for the ease of operations](https://docs.microsoft.com/en-us/azure/frontdoor/front-door-overview) after confirming there is no need to split workspaces due to scale. The app team estimates that the ingestion rate is going to be less than `6GB/minute`, so they expect not to be throttled as this is supported by the default rate limit. If it was required, they could grow by changing this setup eventually. In other words, the design decision is to create a single Azure Log Analytics workspace instance in the `eastus2` region, and that is going to be shared among their multiple clusters. Additionally, there is no business requirement for a consolidated cross business units view at this moment, so the centralization is great option for them. Something the app team also considered while making a final decision is the fact that migrating from a _centralized_ solution to a _decentralized_ one can be much easier than doing it the other way around. As a result, the single workspace being created is a workload specific workspace, and these are some of the Azure services sending data to it:
```

Comment:
> https://docs.microsoft.com/en-us/azure/frontdoor/front-door-overview

Wrong href on that Log Analytics link, yeah?

Reply:
good catch, it is now addressed from [f9e29](https://github.com/mspnp/aks-secure-baseline/pull/172/commits/f9e29e4dbcde5e96c4488344db95491c0181fc59)